### ISSUE TITLE
The Secrets are not passed to github action workflows that are triggered by a pull request from a fork

### ISSUE THREAD

**Issue by larainema** [openmainframeproject/feilong#367]
@mfriesenegger @jmertic 
We have do some investigates, but didn't find a good solution for this issue, need some help and discuss from team and linux foundation. 
This block the github action to access the Vicom infinity Z infrastructure.

**Comment by jmertic** [comment]
@pono Any ideas here?

**Comment by larainema** [comment]
We have proposed one solution:
1. if a pull request from forked repo branch
2. create a local branch from this forked repo branch (might need permission to create local branch)
3. then create a new pull request from the local branch to get the secret

This is the proposal, not test yet, any comments?

**Comment by jichenjc** [comment]
@larainema any thing need help from @Jmetric?I think we need only inform what's need from us and later on with that we can perform the test..

**Comment by mfriesenegger** [comment]
@larainema  I am on the monthly OMP Technical Advisory Council call and it was mentioned by @jmertic that there is a solution so the secrets work properly with GitHub Actions workflows.  Do you know this and have you had a chance to try the solution?

**Comment by larainema** [comment]
> @larainema I am on the monthly OMP Technical Advisory Council call and it was mentioned by @jmertic that there is a solution so the secrets work properly with GitHub Actions workflows. Do you know this and have you had a chance to try the solution?

@mfriesenegger yes, @jmertic mentioned in the slack that pull_request_target event trigger might works, we plan to try this.

**Comment by Bischoff** [comment]
@dvjsharma determined that using `pull_request_target` is indeed the correct method.

On my side, I am always working from a fork and never encountered the issue.

So it looks like the issue has been fixed already. Closing.
